game/tests.py

```
@@ -160,6 +164,30 @@ def test_missing_email_credentials_prints_otp_in_debug(self):
         self.assertIn('Development registration OTP', printed_messages)
         self.assertIn('devplayer@example.com', printed_messages)
 
+    @override_settings(
+        DEBUG=False,
+        EMAIL_HOST_USER='sender@example.com',
+        EMAIL_HOST_PASSWORD='app-password'
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "EMAIL_HOST_PASSWORD"

(S106)